examples/append_conc_to_h5file.py

```diff
@@ -0,0 +1,21 @@
+"""Edit the concentration parameter of a saved HDF5 file
+========================================================
+
+Loads in a previously saved HDF5 dataset. If the attribute
+'concentration' does not exist in the acq_params, this will 
+add the concentration defined by 'actual_conc' into acq_params
+under 'concentration'. If the attribute already exists it will
+simply replace the current concentration with the value of 
+'actual_conc'.
+"""
+from pyspecdata import *
+import h5py,os
+data_dict = {"filename":'220126_Ras_M67R1a_capProbe.h5', #file that is being edited
```

Comment:
Call this data file info or something... It's not a dictionary of the data.

While you're at it, you can use the dict( method of building the dictionary which is a little bit more clear